src/tests/EdFi.DataManagementService.Tests.E2E/Features/EqualityConstraintValidation.feature

```diff
@@ -47,26 +58,36 @@ Feature: Equality Constraint Validation
                       "totalInstructionalTime": 325,
                       "classPeriods": [
                           {
-                          "classPeriodReference": {
-                              "classPeriodName": "01 - Traditional",
-                              "schoolId": 1
-                          }
+                              "classPeriodReference": {
+                                  "classPeriodName": "01 - Traditional",
+                                  "schoolId": 1
+                              }
                           },
                           {
-                          "classPeriodReference": {
-                              "classPeriodName": "02 - Traditional",
-                              "schoolId": 1
-                          }
+                              "classPeriodReference": {
+                                  "classPeriodName": "02 - Traditional",
+                                  "schoolId": 1
+                              }
                           }
                       ],
                       "dates": [],
                       "gradeLevels": []
-                      }
+                  }
                   """
              Then it should respond with 400
               And the response body is
                   """
-                  {"detail":"The request could not be processed. See 'errors' for details.","type":"urn:ed-fi:api:bad-request","title":"Bad Request","status":400,"correlationId":null,"validationErrors":null,"errors":["Constraint failure: document paths $.classPeriods[*].classPeriodReference.schoolId and $.schoolReference.schoolId must have the same values"]}
+                  {
+                      "detail": "The request could not be processed. See 'errors' for details.",
+                      "type": "urn:ed-fi:api:bad-request",
+                      "title": "Bad Request",
+                      "status": 400,
+                      "correlationId": null,
+                      "validationErrors": null,
+                      "errors": [
+                          "Constraint failure: document paths $.classPeriods[*].classPeriodReference.schoolId and $.schoolReference.schoolId must have the same values"
+                      ]
```

Comment:
This is the expected format as found in the ODS/API 7.2 final release. It probably changed after we capture the original message above. Can you create a ticket changing this?

```json
{
  "detail": "Data validation failed. See 'validationErrors' for details.",
  "type": "urn:ed-fi:api:bad-request:data-validation-failed",
  "title": "Data Validation Failed",
  "status": 400,
  "correlationId": "f6172e67-f74e-4723-bd56-8a026ff45580",
  "validationErrors": {
    "$.classPeriods[0].classPeriodReference.schoolId": [
      "All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '255901', '255901001'"
    ]
  }
}
```